ext/sqlite3/sqlite3.stub.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+class SQLite3
+{
+    function __construct(string $filename, int $flags = SQLITE3_OPEN_READWRITE | SQLITE3_OPEN_CREATE, ?string $encryption_key = null);
```

Comment:
`$encryption_key` cannot be `null`, see https://3v4l.org/PDRvd. (Don't rely on the docs; these may be wrong.) In this case the default value is `''`.